### ISSUE TITLE
Incorrect focus order for PDF download link in Chrome on Windows

### ISSUE THREAD

**Issue by lschiff** [eScholarship/jschol#966]
The link to download the PDF article does not appear in the correct focus order when using Chrome on Windows (ex. [https://escholarship.org/uc/item/3962p2f5](https://escholarship.org/uc/item/3962p2f5) ).  The link should be in the focus order before the PDF viewer

This is item 75 in the [accessibility issue report](https://docs.google.com/spreadsheets/d/1kIMiW09tBtk5d3kJjzUMSCo1tEWPn7bUc0BYAJes9Hw/edit?gid=0#gid=0)

Here is the selector: .c-pdfview__accessibility > a:nth-child(1)

**Comment by nicktahani** [comment]
After consulting with Andrew and testing in Browserstack, this appears to be an outdated Chrome bug that is no longer reproducible. Closing.
